ui/cryptomaterial/dropdown.go

```diff
@@ -32,16 +33,26 @@ type DropDown struct {
 	group               uint
 	closeAllDropdown    func(group uint)
 	isOpenDropdownGroup func(group uint) bool
-	Width               int
+	Width               unit.Dp
 	linearLayout        *LinearLayout
 	padding             layout.Inset
 	shadow              *Shadow
+
+	extraDisplay        func(gtx C) D
+	FontWeight          font.Weight
+	BorderWidth         unit.Dp
+	BorderColor         *color.NRGBA
+	NavigationIconColor *color.NRGBA
+	Hoverable           bool
 }
 
 type DropDownItem struct {
 	Text      string
 	Icon      *Image
 	clickable *Clickable
+	// DisplayFn is an alternate display function that conforms to the dropdown
+	// item list display.
+	DisplayFn func(gtx C) D
```

Comment:
```suggestion
	// DisplayFn is an alternate display function that can be used to layout the item
	// instead of using the default item layout.
	DisplayFn func(gtx C) D
```